### PR TITLE
add FURY

### PR DESCRIPTION
**Pull request by cryptofanaticXx1**

### CODE REVIEW

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -630,6 +630,17 @@ const chainInfos = (
...
         },
+        {
+          type: "cw20",
+          contractAddress:
+            "juno1cltgm8v842gu54srmejewghnd6uqa26lzkpa635wzra9m9xuudkqa2gtcz",
+          coinDenom: "FURY",
+          coinMinimalDenom:
+            "cw20:juno1cltgm8v842gu54srmejewghnd6uqa26lzkpa635wzra9m9xuudkqa2gtcz:FURY",
+          coinDecimals: 6,
+          coinGeckoId: "pool:fury",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:641]: Will this Id be defined in the config/prices.ts file? At the moment this Id would do nothing, but if there is a pool, we could then have a price.

File: packages/web/config/chain-infos.ts [hunk 2]
```diff
@@ -630,6 +630,17 @@ const chainInfos = (
           coinGeckoId: "pool:muse",
           coinImageUrl: "/tokens/muse.svg",
         },
+        {
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:633]: Is a PR to the Chain Registry in the works?

Comment (cryptofanaticXx1) [reply packages/web/config/chain-infos.ts:633]: done
https://github.com/cosmos/chain-registry/pull/1056

File: packages/web/config/chain-infos.ts (hunk 1 above)
Comment (cryptofanaticXx1) [reply packages/web/config/chain-infos.ts]: changed the id to https://www.coingecko.com/en/coins/fanfury
"fanfury"
pool is planned, I will define the Id once the pool is created

**Pull request merged**